skbio/stats/power.py

```diff
@@ -173,6 +174,15 @@ def subsample_power(test, samples, draw_mode='ind', alpha_pwr=0.05,
         "ind" mode should be used.
     alpha_pwr : float, optional
         The critical value used to calculate the power.
+    ratio : 1-D array, optional
+        The fraction of the sample counts which should be
+        assigned to each group. If this is a 1-D array, it must be the same
+        length as `samples`. If no value is supplied (`ratio` is None),
+        then an equal number of observations will be drawn for each sample.
+    num_p : unsigned int, optional
```

Comment:
Sorry. I updated the function so `num_p` is calculated automatically (lines 317-324) and then forgot to remove the description from the doc string.